### PR TITLE
Fix IModelBinder docs

### PR DESCRIPTION
**Pull request by rynowak**

### CODE REVIEW

File: src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs [hunk 1]
```diff
@@ -6,22 +6,26 @@
 namespace Microsoft.AspNet.Mvc.ModelBinding
 {
     /// <summary>
-    /// Interface for model binding.
+    /// Defines an interface for model binders.
```
Comment (DamianEdwards) [inline src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs]: Standard for .NET API docs of interfaces usually takes the form "Represents a ..", so in this case e.g. "Represents an object that performs model binding operations.", or some such.

File: src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs [hunk 2]
```diff
@@ -6,22 +6,26 @@
...
         /// <summary>
-        /// Async function to bind to a particular model.
+        /// Attempts to bind a model.
         /// </summary>
-        /// <param name="bindingContext">The binding context which has the object to be bound.</param>
-        /// <returns>A Task which on completion returns a <see cref="ModelBindingResult"/> which represents the result
-        /// of the model binding process.
+        /// <param name="bindingContext">The <see cref="ModelBindingContext"/>.</param>
+        /// <returns>
+        /// <para>
```
Comment (pranavkm) [inline src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs]: Does VS intellisense look ok if the first entry is a `para`?

File: src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs (hunk 1 above)
Comment (rynowak) [inline src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs]: Represents is literally the worst

Comment (DamianEdwards) [inline src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs]: Being inconsistent in our language is worse IMO.

Comment (rynowak) [inline src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs]: So lets be consistent and not use it. Why keep doing something that's awful

File: src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs (hunk 2 above)
Comment (rynowak) [inline src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs]: sure

File: src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs (hunk 1 above)
Comment (DamianEdwards) [inline src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs]: Wow, thanks for volunteering to do the sweep of our code base to ensure we use "Defines" everywhere! :trollface:

Seriously though, you're right. "Defines" is nicer, and we should use it consistently, but we currently don't.

**Pull request closed without merging**